recipes/qtadvanceddocking/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+cmake \
+  -DCMAKE_PREFIX_PATH=${PREFIX} \
```

Comment:
Please add `${CMAKE_ARGS}`. It adds variables defined by conda-forge internally.